packages/tools/viewer-alpha/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require("path");
+const webpackTools = require("@dev/build-tools").webpackTools;
+
+module.exports = (env) => {
+    env = env || {};
+    const source = env.source || "dev";
+    return {
+        // Define the mode as development
+        mode: "development",
+
+        // Entry point of your application
+        entry: "./src/index.ts",
+
+        ...webpackTools.commonDevWebpackConfiguration(
+            {
+                ...env,
+            },
+            {
+                static: ["public"],
+                port: process.env.VIEWER_PORT || 1339,

```

Comment:
Nit - setting port to 1339 would mean that the viewer and the sandbox can't run in parallel when in dev mode (in local environment). You can change it to 1342, which is the next free port. However, feel free to leave it if you know it makes no sense to run both of those At the same time.